designsafe/static/scripts/data-depot/components/projects/pipeline-authors/pipeline-authors.component.js

```diff
@@ -82,7 +82,7 @@ class PipelineAuthorsCtrl {
 
     goWork() {
         window.sessionStorage.clear();
-        this.$state.go('projects.view.data', {projectId: this.project.uuid}, {reload: true});
+        this.$state.go('projects.view', {projectId: this.project.uuid}, {reload: true});
     }
```

Comment:
Good catch, noticed the same issue in `pipeline-licenses.component.js`. Needs the same correction.